integration/tests/cook/util.py

```diff
@@ -15,6 +16,14 @@
 DEFAULT_TIMEOUT_MS = 120000
 
 
+def continuous_integration():
+    return os.environ.get('CONTINUOUS_INTEGRATION')
```

Comment:
What sets this environment variable?

Reply:
[Travis sets it.](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables). We'll probably also want to set it when we run tests internally before a deployment.

Reply:
Added a comment clarifying that Travis sets it automatically.